whatlies/language.py

```diff
@@ -28,16 +28,20 @@ def _input_str_legal(string):
 
     def __getitem__(self, string):
         self._input_str_legal(string)
-        doc = self.nlp(string)
-        vec = doc.vector
+        bert = False
         start, end = 0, -1
         split_string = string.split(" ")
         for idx, word in enumerate(split_string):
             if word[0] == "[":
+                bert = True
```

Comment:
i wonder why these changes were made. could you clarify?

Reply:
technically ... `bert` is but a single type of contextualized embedding ... odds are there will be many other models too.